### PR TITLE
Update the link

### PR DESCRIPTION
**Pull request by joinimran**
As the link was not working so I have updated the link.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10028

### CODE REVIEW

File: windows/deployment/deploy-whats-new.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ The following Delivery Optimization policies are removed in the Windows 10, vers
 - Intune console updates: target version is now available allowing you to specify which version of Windows 10 you want devices to move to. Additionally, this capability enables you to keep devices on their current version until they reach end of service. Check it out in Intune, also available as a Group Policy and Configuration Service Provider (CSP) policy.
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-whats-new.md:92]:
```suggestion
- Intune console updates: Target version is now available allowing you to specify which version of Windows 10 you want devices to move to. Additionally, this capability enables you to keep devices on their current version until they reach end of service. Check it out in Intune, also available as a Group Policy and Configuration Service Provider (CSP) policy.
```

File: windows/deployment/deploy-whats-new.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ The following Delivery Optimization policies are removed in the Windows 10, vers
...
 - Validation improvements: To ensure devices and end users stay productive and protected, Microsoft uses safeguard holds to block devices from updating when there are known issues that would impact that device. Also, to better enable IT administrators to validate on the latest release, we have created a new policy that enables admins to opt devices out of the built-in safeguard holds.
 
-- [**Automatic Restart Sign-on (ARSO)**](/windows-insider/at-work-pro/wip-4-biz-whats-new#automatic-restart-and-sign-on-arso-for-enterprises-build-18305): Windows will automatically log on as the user and lock their device in order to complete the update, ensuring that when the user returns and unlocks the device, the update will be completed.
+- [**Automatic Restart Sign-on (ARSO)**](https://docs.microsoft.com/windows-server/identity/ad-ds/manage/component-updates/winlogon-automatic-restart-sign-on--arso-): Windows will automatically log on as the user and lock their device in order to complete the update, ensuring that when the user returns and unlocks the device, the update will be completed.
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-whats-new.md:95]:
```suggestion
- [**Automatic Restart Sign-on (ARSO)**](/windows-server/identity/ad-ds/manage/component-updates/winlogon-automatic-restart-sign-on--arso-): Windows will automatically log on as the user and lock their device in order to complete the update, ensuring that when the user returns and unlocks the device, the update will be completed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
Ready to go live

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**